cmd/ccli/ccli_test.go

```diff
@@ -15,37 +15,80 @@
 package ccli
 
 import (
+	"context"
+	"crypto/tls"
 	"flag"
+	"os"
 	"strings"
 	"testing"
 
-	log "github.com/golang/glog"
 	"github.com/openconfig/gribigo/compliance"
 	"github.com/openconfig/gribigo/fluent"
 	"github.com/openconfig/gribigo/negtest"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
+
+	spb "github.com/openconfig/gribi/v1/proto/service"
 )
 
 var (
-	addr = flag.String("addr", "", "address of the gRIBI server in the format hostname:port")
+	addr       = flag.String("addr", "", "address of the gRIBI server in the format hostname:port")
+	insecure   = flag.Bool("insecure", false, "dial insecure gRPC (no TLS)")
+	skipverify = flag.Bool("skipverify", true, "allow self-signed TLS certificate")
```

Comment:
(nit) Add something here that says that this is not required when "insecure" is also set?

Reply:
Good idea! Done.